main/src/ca/uwaterloo/flix/language/phase/unification/BddFormula.scala

```diff
@@ -103,10 +105,10 @@ object BddFormula {
     override def minimize(f: DD): DD = f
 
     override def toType(f: DD, env: Bimap[Symbol.KindedTypeVarSym, Int]): Type = {
-      createTypeFromBDDAux(f, Type.True, env)
+      toTypeQMC(f, env)
```

Comment:
I added a flag `--Xqmc`. Available in the Flix `options` object.

Can we use it to toggle whether to enable QMC?

Reply:
(You need to rebase/merge onto master).